internal/it/util.go

```diff
@@ -61,6 +61,7 @@ var defaultDedicatedTestCluster = NewSingletonTestCluster(defaultDedicatedCluste
 	port := NextPort()
 	return rc.startNewCluster(MemberCount(), XMLConfig(defaultDedicatedClusterName, port), port)
 })
+var DefaultDedicatedTestCluster = defaultDedicatedTestCluster
```

Comment:
I think this change is not related to this PR

Reply:
Removed at: https://github.com/hazelcast/hazelcast-commandline-client/pull/374/commits/8d957cf473ec7d3b64ffb964f223c176feea2d3c